src/adaptors/index-coop/index.js

```diff
@@ -0,0 +1,36 @@
+const { default: BigNumber } = require('bignumber.js');
+const utils = require('../utils');
+
+const buildPool = (index, apy) => {
+  const chain = utils.formatChain(index.chain);
+  const symbol = utils.formatSymbol(index.symbol);
+  return {
+    pool: `${index.address}-${chain}`.toLowerCase(),
+    chain,
+    project: 'index-coop',
+    symbol,
+    tvlUsd: 0,
+    apy,
+  };
+};
+
+const main = async () => {
+  const icEthIndex = {
+    address: '0x7C07F7aBe10CE8e33DC6C5aD68FE033085256A84',
+    chain: 'Ethereum',
+    symbol: 'icETH',
+  };
+
+  // Will only work from defillama.com domain (others can be requested)
+  const { apy } = await utils.getData('https://api.indexcoop.com/iceth/apy');
```

Comment:
As the endpoint requires auth, please let us know what domains to put on the allowlist. For now it's defillama.com only.

Reply:
we query it from a server

Reply:
also pls get data from on-chain instead of apis